### ISSUE TITLE
Add oidc authorization support

### ISSUE THREAD

**Issue by kvaps** [splunk/qbec#14]
Hi, when I try to run `qbec apply` command, I have an error:

```
✘ No Auth Provider found for name "oidc"
```

version:
```
qbec version: 0.6.1
jsonnet version: v0.11.2
go version: 1.10.3
commit: a5dd69e
```

**Comment by gotwarlost** [comment]
I'll look into the oidc auth. In the meantime can you try this option?

https://kubernetes.io/docs/reference/access-authn-authz/authentication/#option-2-use-the-token-option

You can pass in the token as `--k8s:token=<idtoken>` in the qbec command

**Comment by kvaps** [comment]
Hi, option is not working same error here

**Comment by gotwarlost** [comment]
@kvaps could you build from source and verify that this fixes your problem? I'll create a release once you have done this.

**Comment by kvaps** [comment]
Hi, now it is working fine, thank you!
```
qbec version: 0.6.2
jsonnet version: v0.11.2
go version: 1.12.1
commit: f51dae7
```

**Comment by gotwarlost** [comment]
Thanks! Just released [v0.6.3](https://github.com/splunk/qbec/releases/tag/v0.6.3) with this fix.
